pkg/controller/plan/kubevirt.go

```diff
@@ -1835,6 +1832,103 @@ func (r *KubeVirt) EnsurePersistentVolume(vmRef ref.Ref, persistentVolumes []cor
 	return
 }
 
+func (r *KubeVirt) CreatePvcForNfs(pvcName string) (err error) {
+	//TODO: set ownerReferenceso the PV and PVC deleted once the plan is done.
+	// ownerReference := meta.OwnerReference{
+	// 	APIVersion: "forklift.konveyor.io/v1beta1",
+	// 	Kind:       "Plan",
+	// 	Name:       r.Plan.Name,
+	// 	UID:        r.Plan.UID,
+	// }
+
+	sourceProvider := r.Source.Provider
+	pvName := fmt.Sprintf("ova-server-pv-%s", r.Plan.Name)
```

Comment:
i would put this line below, right before the `pv := &...` and let the error run first